src/api/weeatApi.js

```diff
@@ -0,0 +1,5 @@
+import axios from 'axios';
+
+export default axios.create({
+   baseURL: 'http://localhost:3000' 
```

Comment:
It's better to have here an environment variable instead of hard-coding this constant. Since you decided using CRA (create-react-app), you can achieve that by creating [environment variables](https://create-react-app.dev/docs/adding-custom-environment-variables).

Reply:
Done